### PR TITLE
add: `grub-btrfs`

### PR DESCRIPTION
**Pull request by erenfro**
Include btrfs snapshots in GRUB boot options

### CODE REVIEW

**Comment by erenfro** [comment]
This is fully tested and ready for merging.

File: packages/grub-btrfs/grub-btrfs.pacscript [hunk 1]
```diff
@@ -0,0 +1,22 @@
+pkgname=grub-btrfs
+pkgver=4.13
+pkgdesc='Include btrfs snapshots in GRUB boot options'
+arch=('any')
```
Comment (Elsie19) [inline packages/grub-btrfs/grub-btrfs.pacscript:4]:
```suggestion
```

`arch=('any')` is implicit if not included.

File: packages/grub-btrfs/grub-btrfs.pacscript [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+url="https://github.com/Antynea/${pkgname}"
```
Comment (Elsie19) [inline packages/grub-btrfs/grub-btrfs.pacscript:5]:
```suggestion
url='https://github.com/Antynea/grub-btrfs'
```

**Review by Elsie19 — CHANGES_REQUESTED** [review]

File: packages/grub-btrfs/grub-btrfs.pacscript [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+pkgver=4.13
+pkgdesc='Include btrfs snapshots in GRUB boot options'
+url='https://github.com/Antynea/grub-btrfs'
+license=('GPL-3.0-only')
+depends=('btrfs-progs' 'grub2-common')
+optdepends=(
+  'snapper: For snapper support'
+  'inotify-tools: For grub-btrfsd daemon'
+)
+backup=('etc/default/grub-btrfs/config')
```
Comment (Elsie19) [inline packages/grub-btrfs/grub-btrfs.pacscript:11]: anything in /etc automatically is considered a config file, so this like is redundant.

**Review by Elsie19 — CHANGES_REQUESTED** [review]

**Review by oklopfer — APPROVED** [review]

**Pull request merged**